src/fundus/publishers/de/sportschau.py

```diff
@@ -0,0 +1,43 @@
+import datetime
+from typing import List, Optional
+
+from lxml.cssselect import CSSSelector
+
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute
+from fundus.parser.utility import (
+    extract_article_body_with_selector,
+    generic_author_parsing,
+    generic_date_parsing,
+)
+
+
+class SportSchauParser(ParserProxy):
+    class V1(BaseParser):
+        _paragraph_selector = CSSSelector("article >p.textabsatz")
```

Comment:
As of now, this selector also include the summary. I would suggest extending this selector by `article >p.textabsatz:not(:nth-of-type(1))`